playground/frontend/playground_components/tools/extract_symbols_java/src/main/java/com/playground/extract_symbols/Main.java

```diff
@@ -61,6 +63,10 @@ private static HashMap<String, ClassInfo> getDirSymbols(String sdkPathString) th
         return classInfoMap;
     }
 
+    static boolean isJavaNonTestFile(String stringPath) {
+        return stringPath.endsWith(".java") && !stringPath.contains("/test/");
```

Comment:
1. Add a non-`.java` file with a public class and a public member to make sure it is ignored.
2. Add `test` folder with a public class and a public member to make sure it is ignored.

Reply:
done